CMakeLists.txt

```diff
@@ -202,6 +214,8 @@ target_include_directories(${PROJECT_NAME}
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
         $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
         $<INSTALL_INTERFACE:include>
+        $<$<BOOL:${EXTERNAL}>:${EXTERNAL_TRANSPORT_HEADER_DIRECTORY}>
+        $<$<BOOL:${EXTERNAL}>:${PROJECT_SOURCE_DIR}/src/c/profile/transport/serial>
```

Comment:
Why don't you use neither `BUILD_INTERFACE` nor `INSTALL_INTERFACE`?

Reply:
Done with `BUILD_INTERFACE`